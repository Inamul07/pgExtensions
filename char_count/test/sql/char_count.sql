CREATE EXTENSION char_count;
SELECT char_count('aaaabbbbbbbcc','a');
SELECT char_count('aaaabbbbbbbcc','b');
SELECT char_count('aaaabbbbbbbcc','c');
SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','x');
SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','c');
SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','b');
SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','5');
SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','3');
SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','2');
SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','1');
SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','0');
SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','asd');
