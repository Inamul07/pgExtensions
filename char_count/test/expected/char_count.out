CREATE EXTENSION char_count;
SELECT char_count('aaaabbbbbbbcc','a');
 char_count 
------------
          4
(1 row)

SELECT char_count('aaaabbbbbbbcc','b');
 char_count 
------------
          7
(1 row)

SELECT char_count('aaaabbbbbbbcc','c');
 char_count 
------------
          2
(1 row)

SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','x');
 char_count 
------------
          0
(1 row)

SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','c');
 char_count 
------------
          2
(1 row)

SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','b');
 char_count 
------------
          7
(1 row)

SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','5');
 char_count 
------------
          5
(1 row)

SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','3');
 char_count 
------------
          7
(1 row)

SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','2');
 char_count 
------------
          7
(1 row)

SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','1');
 char_count 
------------
          4
(1 row)

SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','0');
 char_count 
------------
          1
(1 row)

SELECT char_count('aaaabbbbbbbcc1111222222233333335555590','asd');
ERROR:  value too long for type character(1)
CONTEXT:  PL/pgSQL function char_count(text,character) line 7 during statement block local variable initialization
